build_tag.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -exvo pipefail -o nounset
+
+# Simple script to download and run goreleaser
+# uses config from .goreleaser.yml
+curl -sL https://git.io/goreleaser | bash
```

Comment:
I'm a bit confused as to how this will work.

How does it know which binaries to build? I don't see it mentioned in the `.goreleaser.yml`.

Does this also publish the binary against an already existing release, or does it create a new release for you? app-intercace's `gh-build-tag` is triggered only upon creating a new release tag.

Reply:
I need to cleanup `.goreleaser.yml`. It is missing instructions on where to find the binary `cmd/mtcli/main.go` and probably other stuff. I've only copied a base file for now but this is indeed incomplete. 

You are right `gh-build-tag` is triggered after pushing a new tag. So whenever we want to trigger a release, we need to manually tag and push. You can see an example here: 
- https://github.com/app-sre/aws-resource-exporter/blob/master/build_tag.sh
- https://github.com/app-sre/aws-resource-exporter/tags

Reply:
Fixed and made sure goreleaser works as expected:
```bash
$ git tag v0.0.1
$ goreleaser check
$ goreleaser release --skip-publish
$ ./dist/mtcli_linux_amd64/mtcli version
```